Changelog.md

```diff
@@ -3,6 +3,7 @@
 Language Features:
  * Allow to obtain the selector of public or external library functions via a member ``.selector``.
  * Parser: Allow splitting string and hexadecimal string literals into multiple parts.
+ * Inline Assembly: Support referencing another constants in inline assembly.
```

Comment:
I think this should be `Support referencing another constant` or `Support referencing other constants`

Reply:
Yeah, I've noticed that "another constants" doesn't sound right. Changed it to "other constants" here https://github.com/ethereum/solidity/pull/7874/commits/b37cbfa16cf313df668bb3ec8af93aed0eeca1ae. I'll change it in the commit message as well, later when I squash the commits.

Reply:
Thank you for your contribution @ghallak :-)